content/02.main-text.md

```diff
@@ -193,6 +195,21 @@ Additional research studies in progress are being authored using Manubot, spanni
 Manubot is also being used for documents beyond traditional journal publications, such as [grant proposals](https://greenelab.github.io/manufund-2018/), [progress reports](https://greenelab.github.io/czi-hca-report/), [undergraduate research reports](https://zietzm.github.io/Vagelos2017/) [@doi:10.6084/m9.figshare.5346577], [literature reviews](https://slochower.github.io/synthetic-motor-literature/), and lab notebooks.
 Finally, manuscripts written with other authoring systems have been successfully ported to Manubot, including the [Bitcoin Whitepaper](https://git.dhimmel.com/bitcoin-whitepaper/) [@tag:steemit-post] and [Project Rephetio manuscript](https://git.dhimmel.com/rephetio-manuscript/) [@doi:10.7554/eLife.26726].
 
+### Citation utility
+
+To make citation-by-identifier easily usable outside of Manubot manuscripts, we created the `manubot cite` command line utility, available as a [Python package](https://pypi.org/project/manubot/).
+This utility takes a list of citations and returns either a rendered bibliography or CSL Data Items (i.e. JSON-formatted reference metadata).
+For example, the following command outputs a Markdown reference list for the two specified articles according to the bibliographic style of _PeerJ_:
+
+```sh
+manubot cite --render --format=markdown \
+  --csl=https://github.com/citation-style-language/styles/raw/master/peerj.csl \
+  pmid:29618526 doi:10.1038/550143a
+```
+
+If the user does not specify a CSL style, the utility defaults to using the Manubot style, which focuses on information relevant for modern scholarly workflows and omits the rest.
```

Comment:
The phrase `information relevant for modern scholarly workflows and omits the rest` seems quite subjective and isn't well-supported in this paragraph.  If there was widespread agreement about what information is relevant, we would have more homogeneous citation styles.  If we are arguing that the choices in Manubot's CSL are innovative and better than other options, we should explain that more explicitly, as your linked blog does for numeric citations.

This information could be briefly integrated into other parts of the manuscript.  In a section above we have
> As a result, users can choose from thousands of existing CSL styles or use Manubot's default style.

If the numeric citations are important, we could change that to
> As a result, users can choose from thousands of existing CSL styles or use Manubot's default style, which emphasizes titles instead of authors and numeric-style citations. [@url:https://blog.dhimmel.com/citation-styles/].

Reply:
I agree with @agitter. I've read the blog post and it makes the case that both styles have advantages. I prefer @agitter's final version.

Reply:
See https://github.com/greenelab/meta-review/pull/99/commits/ee1a34760e6433afc6a9e8eef95fbe07a6aa987f. I realized there was some redundancy that had crept into the CSL style paragraph, so I addressed that as well.